Components/Transceivers/RDMA/RDMA Windows.lvproj

```diff
@@ -376,7 +759,13 @@
 				<Property Name="Source[1].preventRename" Type="Bool">true</Property>
 				<Property Name="Source[1].sourceInclusion" Type="Str">TopLevel</Property>
 				<Property Name="Source[1].type" Type="Str">Library</Property>
-				<Property Name="SourceCount" Type="Int">2</Property>
+				<Property Name="Source[2].destinationIndex" Type="Int">0</Property>
+				<Property Name="Source[2].itemID" Type="Ref">/My Computer/Documents/dsf-RDMA-errors.txt</Property>
+				<Property Name="Source[2].sourceInclusion" Type="Str">Include</Property>
+				<Property Name="Source[3].destinationIndex" Type="Int">0</Property>
```

Comment:
The files need to be explicitly added to the Errors directory.
![image](https://user-images.githubusercontent.com/29306186/175579446-aabbbfd8-6e65-48c2-86f3-41b5e01c7e02.png)

Reply:
I'm just not functioning properly today. I've made the change.